cmd/state-svc/internal/resolver/resolver.go

```diff
@@ -170,6 +171,17 @@ func (r *Resolver) RuntimeUsage(ctx context.Context, pid int, exec string, dimen
 	return &graph.RuntimeUsageResponse{Received: true}, nil
 }
 
+func (r *Resolver) RuntimeAttempt(ctx context.Context, exec string, dimensionsJSON string) (*graph.RuntimeAttemptResponse, error) {
```

Comment:
This endpoint isn't needed, we already reach out to the svc for analytics. Just call the analytics event from the async client in the state tool.

The reason heartbeats have their own endpoint is because it needs to do things beside just send an analytics event.